### DISCUSSION TITLE
Sign smart contract tx error

### DISCUSSION THREAD

**Discussion by ghost** [CoolBitX-Technology/coolwallet-signing-tools#38] [Q&A] [answered]
- When we run `signSmartContractTransaction` function in `coint eth` with newest script generated by signing tools to test interact with contract in ether network. 
- Script : 03050601C707000000003CA00700C2ACD70032FFF8C2ACD7001EFFF6C2ACD70028FFF6CC071094CAA02700C2A2D700FFF6C4ACC7003A04CC0E1001C2E09700CC07C0028080BE0710DC07C003455448D207C005534D415254D207CC05065052455353425554546F4EFA0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
- Arguments:  15328000002C8000003c80000000000000000000000050d1c9771902476076ecfc8b2a83ad6b9355a4c90000000000000000000000000000000077359401000000000000000493e0000000000000000200000024
- Expect a success signed transaction but sdk throw this error:
![image](https://user-images.githubusercontent.com/85855238/150258418-3b0f6781-13c9-4ddd-b525-7acc2724e8f1.png)

Can you check sign smart contract tx function and script 
(We think error happend because version of script is `05` and we haven't upgrade cool wallet version yet. )

**Comment by hank121314** [comment, marked as answer]
Would you mind pulling the latest script-tester and updating the coolwallet version again? Thanks!

**Reply by ghost** [reply]
It's worked, thank sir
